src/lib/pages/backoffice/Loan/LoanDetails/AvailableItems/AvailableItems.js

```diff
@@ -66,23 +72,54 @@ export default class AvailableItems extends Component {
     );
   }
 
+  performCheckout(loan, item = null) {
+    const { performCheckoutAction } = this.props;
+    const { checkoutItem, modalOpen } = this.state;
+    item = item ? item : checkoutItem;
+    if (modalOpen) {
+      this.setState({ modalOpen: !modalOpen });
+    }
+    const checkoutUrl = loan.availableActions.checkout;
+    const patronPid = loan.metadata.patron_pid;
+    const documentPid = item.metadata.document.pid;
+    const itemPid = {
+      type: recordToPidType(item),
+      value: item.metadata.pid,
+    };
+    performCheckoutAction(checkoutUrl, documentPid, patronPid, itemPid);
+  }
+
+  handleCloseModal = () => {
+    this.setState({ modalOpen: false });
+  };
+
   checkoutItemButton(item, loan) {
-    const { performCheckoutAction, isActionLoading } = this.props;
+    const { documentPendingLoans, isActionLoading } = this.props;
+    documentPendingLoans.hits.sort((a, b) => {
+      return (
+        new Date(a.metadata.request_start_date) -
```

Comment:
I am not sure if we should use the `metadata.request_start_date` instead the `created` for sorting the array. I did it like this because it could be that someone created a request that will start a month later, given this situation I understand that we should order by the `request_start_date`

Reply:
I think you should let Elasticsearch sort, so in the search query, add the `sort` param and the right `field` and `order`.